ocaml/typing/primitive.mli

```diff
@@ -29,6 +29,12 @@ type description = private
   { prim_name: string;         (* Name of primitive  or C function *)
     prim_arity: int;           (* Number of arguments *)
     prim_alloc: bool;          (* Does it allocates or raise? *)
+    prim_c_builtin: bool;
+    (* When [prim_builtin] is true, the native compiler is allowed to rewrite
```

Comment:
The comment needs to be updated to use `prim_c_builtin` not `builtin`.  Also it would be better if the parameter to `make` below was called `c_builtin`.  Likewise for the various other occurrences of `builtin` in this patch.

Reply:
done.